src/pages/productListing/components/filter/filter.js

```diff
@@ -1,24 +1,39 @@
-import axios from "axios";
+import { useProduct } from "../../../../context/products/productContext";
+import { ratings } from "../../../../temp/rating.json";
 
 const Filter = () => {
+  let { state, categories, dispatch } = useProduct();
+
   return (
     <div className="product-sidebar">
       <div className="filters-container mx-2">
         <div className="filter-type flex-row justify-between align-center my-2">
           <p className="mr-1 text-sm font-bold">FILTERS</p>
-          <a href="/">
-            <span className="text-xxs">CLEAR ALL</span>
-          </a>
+          <button
+            className="text-xxs"
+            value="RESET_FILTER"
+            onClick={(e) =>
```

Comment:
Suggestion: I think value attribute could be removed

Reply:
removed !!